src/App.jsx

```diff
@@ -14,57 +16,81 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const startIndex = goods.findIndex(products => products === 'Jam');
+  const [activeRow, setActiveRow] = useState(startIndex);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  const [message, setMessage] = useState('Jam');
+  const [isButtonVisible, setIsButtonVisible] = useState(true);
+  const RemoveButton = () => {
+    setMessage('No goods');
+    setActiveRow(null);
```

Comment:
`ClearButton should clear the selection by setting an empty string to selectedGood.
`
add also **setMessage(EMPTY_MESSAGE)**